### PR TITLE
redirect supervisor to sessions log

### PR DESCRIPTION
**Pull request by ericcf**
[#118408339]

### CODE REVIEW

File: app/controllers/supervision_sessions_controller.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def create
     authorize! :create, @supervision_session
 
     if @supervision_session.save
-      redirect_to nurse_supervisor_dashboard_url,
+      redirect_to nurse_supervision_sessions_url(@nurse),
```
Comment (michaelwehrley) [inline app/controllers/supervision_sessions_controller.rb]: Did this ever work?

**Comment by michaelwehrley** [comment]
👍 🚀

File: app/controllers/supervision_sessions_controller.rb (hunk 1 above)
Comment (ericcf) [inline app/controllers/supervision_sessions_controller.rb]: Yes, they just changed their minds about where it should redirect.

Comment (michaelwehrley) [inline app/controllers/supervision_sessions_controller.rb]: Gosh, I totally misread it.  Got it.  Thanks.

**Pull request merged**